sensor_anomalies.tex

```diff
@@ -50,8 +50,8 @@ \subsubsection{Crosstalk}
 \subsubsection{Bleed Trails}
 Bleed trails from saturated sources were expected on LSSTComCam, but they appear in more dramatic forms than was expected.
 As a bleed trail nears the serial register, it fans out into a ``trumpet'' shaped feature.
-Although bright, these features do not have consistently saturated pixels, and were ignored in the first on-sky processing.
-We have since developed the means to programmatically identify and mask these features, which we have named ``edge bleeds.''
+Although bright, these features do not have consistently saturated pixels.
+In DP1 these ``edge bleeds'' were programmatically identified and masked.
```

Comment:
Talking about problems that existed before and were then solved before DP1 obfuscates what was actually solved in time for DP1.  At least I think we consider this fully solved in DP1?  @aferte, can you confirm?